docs-new/docs/graphql/core/databases/postgres/schema/table-relationships/create.mdx

```diff
@@ -1,17 +1,23 @@
-<div class="meta"
-description="Create relationships between Postgres tables/views in Hasura"
-keywords="hasura, docs, postgres, schema, relationship, create">
-
-</div>
+---
+sidebar_label: Creating relationships
+sidebar_position: 1
+description: Create relationships between Postgres tables/views in Hasura
+keywords:
+  - hasura
+  - docs
+  - postgres
+  - schema
+  - relationship
+  - create
+---
+
+import GraphiQLIDE from '@site/src/components/GraphiQLIDE/GraphiQLIDE';
+import Thumbnail from '@site/src/components/Thumbnail/Thumbnail';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
 # Postgres: Creating relationships
 
-<div class="contents" backlinks="none" depth="1" local="">
-
-Table of contents
-
-</div>
-
 ## Introduction
 
 A relationship from one table/view to another can be created by defining
```

Comment:
adding here as unable to add comment in Line 32

missing custom header id

`## Using foreign keys`

Reply:
Done